clients/analysis/src/__tests__/__resources__/cfgraph/case121.result.dot

```diff
@@ -0,0 +1,7 @@
+digraph A {
+rankdir=LR;
+<PROG>-><MAIN-LOGIC>;
+<PROG>-><CLEAN-UP>;
```

Comment:
I know this is how it is actually implemented in a real program, but is it intentional that we present the abend call as a dead end without a node that just magically appears at the handle label call site?
Should the exec cics abend instruction have its own node "called" from paragraph-b in this case?

Reply:
Good idea. I suggest to merge this as it is and I will add a support for terminator nodes (STOP RUN, EXIT PROGRAM, GOBACK, etc.) so frontend can show them with different node style